vars/folioHelm.groovy

```diff
@@ -128,31 +129,114 @@ void checkPodRunning(String ns, String podName) {
   }
 }
 
-void checkAllPodsRunning(String ns) {
-  timeout(time: ns == 'ecs-snapshot' ? 20 : 10, unit: 'MINUTES') {
-    boolean notAllRunning = true
-    while (notAllRunning) {
-      sleep(time: 30, unit: 'SECONDS')
+//void checkAllPodsRunning(String ns) {
+//  timeout(time: ns == 'ecs-snapshot' ? 20 : 10, unit: 'MINUTES') {
+//    boolean notAllRunning = true
+//    while (notAllRunning) {
+//      sleep(time: 30, unit: 'SECONDS')
+//
+//      def result = sh(script: "kubectl get pods -n ${ns} --no-headers | awk '{print \$3}'", returnStdout: true).trim()
+//
+//      notAllRunning = result.split('\n').any { status -> status != 'Running' }
+//
+//      if (notAllRunning) {
+//        def evictedPodsList
+//        println('Not all pods are running. Retrying...')
+//        try {
+//          evictedPodsList = sh(script: "kubectl delete pod -n ${ns} --field-selector=\"status.phase==Failed\"", returnStdout: true)
+//        } catch (Error err) {
+//          new Logger(this, "managePods").warning("Error: " + err.getMessage() + "\nList of evicted pods: ${evictedPodsList}")
+//        }
+//      } else {
+//        println('All pods are running.')
+//      }
```

Comment:
This section with evictense should be presented in our code. Possibly we could write it as a separate function